maintainer.sh

```diff
@@ -1,16 +1,25 @@
 #!/bin/sh
-set -x
-
 aclocal # Set up an m4 environment
 autoconf # Generate configure from configure.ac
 automake --add-missing # Generate Makefile.in from Makefile.am
 
 #Test configure & build
-./configure --prefix=$PWD CC=clang
+./configure CC=clang
```

Comment:
Any particular reason we're using clang?

Reply:
Original makefile used clang. I've pushed a change removing that.